src/main/java/hudson/maven/reporters/AbstractMavenJavadocArchiver.java

```diff
@@ -46,12 +46,6 @@
  */
 public abstract class AbstractMavenJavadocArchiver extends MavenReporter {
 
-
-    private boolean aggregated = false;
```

Comment:
This field was never read outside of the method in which it was defined, so why was it a field to begin with?

Reply:
Mark as deprecated? CC @aheritier @olamy